public/app/percona/dbaas/components/Kubernetes/KubernetesInventory.tsx

```diff
@@ -53,20 +60,44 @@ export const KubernetesInventory: FC<KubernetesInventoryProps> = ({ setMode }) =
   const { result: kubernetes, loading: kubernetesLoading } = useSelector(getKubernetesSelector);
   const { loading: deleteKubernetesLoading } = useSelector(getDeleteKubernetes);
   const { loading: addKubernetesLoading } = useSelector(getAddKubernetes);
-  const loading = kubernetesLoading || deleteKubernetesLoading || addKubernetesLoading;
+  const [update, setUpdate] = useState(false);
+  const [catchFromAsyncThunkAction] = useCatchCancellationError();
+
+  const loading = (kubernetesLoading || deleteKubernetesLoading || addKubernetesLoading) && !update;
+  const k8sListShouldBeUpdated = kubernetes && kubernetes.find((item) => item.status === K8SStatus.provisioning);
 
   const deleteKubernetesCluster = useCallback(
     async (force?: boolean) => {
       if (selectedCluster) {
         setDeleteModalVisible(false);
-        await dispatch(deleteKubernetesAction({ kubernetesToDelete: selectedCluster, force }));
+        await appDispatch(deleteKubernetesAction({ kubernetesToDelete: selectedCluster, force }));
         setSelectedCluster(null);
       }
     },
     // eslint-disable-next-line react-hooks/exhaustive-deps
     [selectedCluster]
   );
 
+  const updateK8Clusters = useCallback(
+    async (triggerLoading = true) => {
+      const result = await catchFromAsyncThunkAction(
+        appDispatch(
+          fetchKubernetesAction({
+            kubernetes: generateToken(GET_KUBERNETES_CANCEL_TOKEN),
+            operator: generateToken(CHECK_OPERATOR_UPDATE_CANCEL_TOKEN),
+          })
+        )
+      );
+
+      // undefined means request was cancelled
```

Comment:
Does this piece of code need to be here? It looks like you aren't doing anything with the result.

Reply:
Yes, thanks! I agree. It was another function previously and i missed this.